protocol/protobuf/proto/state_representation/trajectory/trajectory.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package state_representation.proto;
+
+import "state_representation/state.proto";
+
+message DoubleArray {
+  repeated double values = 1;
+}
+
+message CartesianTrajectory {
+  State state = 1;
+  repeated DoubleArray data = 2;
+  repeated string names = 3;
+  repeated uint64 durations = 4;
+  string reference_frame = 5;
+}
+
+message JointTrajectory {
+  State state = 1;
+  repeated DoubleArray data = 2;
+  repeated string names = 3;
+  repeated uint64 durations = 4;
+  repeated string joint_names = 5;
+}
```

Comment:
do you have an explicit preference for `reference_frame` and `joint_names` to be the last field and not the second field? It doesn't make much of a difference but when you are encoding / decoding you are always going in the order of state name, reference / joint names, and then the data, names and durations. It's nice to keep some kind of "convention" in the order if possible.

Reply:
I chose to have least to most specialized variable order, but that was as far as my preference went. I agree that during parsing this is actually not reflected because we actually need to access the specialized parts first, so makes sense, thanks!